src/test/java/cloudBankTestCases/ClientCreationTestCase.java

```diff
@@ -0,0 +1,99 @@
+package cloudBankTestCases;
+
+import cloudBankPages.ClientListPage;
+import cloudBankPages.CreateClientPage;
+import cloudBankPages.Homepage;
+import cloudBankPages.LogInPage;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.time.Duration;
+
+import static org.testng.Assert.assertTrue;
+
+public class ClientCreationTestCase {
+
+    WebDriver driver;
+
+    LogInPage objLogInPage;
+    Homepage objHomePage;
+    CreateClientPage objCreateClientPage;
+    ClientListPage objClientListPage;
+
+    String clientAccountNum;
+
+    @BeforeTest
+    void setUp() {
+
+        //chrome driver setup
+        String projectPath = System.getProperty("user.dir");
+        System.setProperty("webdriver.chrome.driver", projectPath+ "/src/test/resources/chromedriver.exe");
+
+        ChromeOptions options = new ChromeOptions();
+        options.setBinary("C:\\Users\\Kamille\\Chrome For Testing\\chrome-win64\\chrome.exe");
+        driver = new ChromeDriver(options);

```

Comment:
Can you try without applying any options? I think the latest version of Selenium will install a browser if it is not present.